lib/events.js

```diff
@@ -351,7 +351,7 @@ EventEmitter.prototype.prependOnceListener =
       return this;
     };
 
-// emits a 'removeListener' event iff the listener was removed
+// emits a 'removeListener' event if and only if the listener was removed
```

Comment:
so it would read:
emits -> Emits a 'removeListener' event if and only if the listener was removed.
??